src/main/kotlin/nl/info/zac/app/zaak/ZaakRestService.kt

```diff
@@ -640,6 +641,11 @@ class ZaakRestService @Inject constructor(
         afbrekenGegevens: RESTZaakAfbrekenGegevens
     ) {
         val zaak = zrcClientService.readZaak(zaakUUID)
+        if (zaak.resultaat != null) {
+            throw CaseWithADecisionCannotBeTerminatedException(
+                "De zaak kan niet afgebroken worden als er een besluit aan hangt."
+            )
+        }
```

Comment:
```suggestion
       zaak.resultaat?.run {
    throw CaseWithADecisionCannotBeTerminatedException(
        "De zaak kan niet afgebroken worden als er een besluit aan hangt."
    )
}
```

More Kotlin idiomatic. Explicit null checks are hardly ever needed and not common in Kotlin.